### PR TITLE
Reorg auction docs

### PR DESCRIPTION
**Pull request by PiVortex**
Reorg auction docs as per https://github.com/near-examples/auctions-tutorial/pull/16

I am happy for this tutorial to get a final review now and go out, I have added everything to the sidebar. The only thing I'm not too keen on is the length of the ft page but I don't know how else to slim it down or split it up - maybe someone has an idea. All the code references are linking to reorg-auction, once auction rework PR is merged this will need changing to main

### CODE REVIEW

**Comment by PiVortex** [comment]
@race-of-sloths

**Comment by gagdiez** [comment]
@race-of-sloths score 12

**Review by gagdiez — APPROVED** [review]

**Comment by race-of-sloths** [comment]
@PiVortex Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
You are true leader! Other sloths hot on your heels, keep moving!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/PiVortex?type=bot&pr=near/docs/2250&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/PiVortex?type=bot&pr=near/docs/2250&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/PiVortex)


<details><summary>Current status: <i>executed</i></summary>

| Reviewer | Score |
|--------|--------|
| @gagdiez  | 13 |


Your contribution is much appreciated with a final score of 13!
You have received 155 (130 base + 15 weekly bonus + 10 monthly bonus) Sloth points for this contribution


Congratulations @PiVortex! Your PR was highly scored and you completed another monthly streak! To keep your monthly streak make another pull request next month and get 8+ score for it
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/PiVortex)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Pull request merged**
